servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcServerBuilder.java

```diff
@@ -134,7 +192,12 @@ public abstract GrpcServerBuilder enableWireLogging(String loggerName, LogLevel
      *
      * @param transportObserver A {@link TransportObserver} that provides visibility into transport events.
      * @return {@code this}.
+     * @deprecated Call {@link #initializeHttp(HttpInitializer)} and use
+     * {@link HttpServerBuilder#transportObserver(TransportObserver)}
+     * on the {@code builder} instance by implementing {@link HttpInitializer#initialize(HttpServerBuilder)}
+     * functional interface.
      */
+    @Deprecated
     public abstract GrpcServerBuilder transportObserver(TransportObserver transportObserver);
```

Comment:
Can not comment there, will leave here:
`lifecycleObserver` is the only method that also exists at `HttpServerBuilder`. While you are here, please clarify in javadoc that if users use `lifecycleObserver(GrpcLifecycleObserver)`, using a similar method `HttpServerBuilder#lifecycleObserver(HttpLifecycleObserver)` will override the value.